package/autoyast2.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Wed Jul  6 16:56:38 CEST 2016 - schubi@suse.de
+
+- If the user changes the path of the AutoYaST configuration file
+  in the UI (installation workflow)  the path will be updated in
```

Comment:
I don't get what `installation workflow` means (maybe it's just me).

np: please, remove the extra space before `the path will`.